Remove login to raccess text

p-galligan opened the pull request:
Removes login to raccess text from request modal.

Additionally, reduces the size of the modal given less content.

Fixes #74

Commit:
Remove login to raccess text

Removes login to raccess text from request modal.

Additionally, reduces the size of the modal given less content.

HaSistrunk commented:
Cool. Looks like you can remove the `modal__info` class from the scss now as well since it's not being used.

helrond commented on _sass/custom.scss:
```diff
@@ -131,7 +131,7 @@ h2:not(:first-child) {
   &.is-open {
     display: block;
-    height: 100%;
+    height: 85%;
```
I don't think this is the right place to change height - that's the background which should always be at 100%

Either try tweaking the `modal-content` class OR just add some padding/margins to the elements in the modal (which actually gives things a bit more room to breathe)

Commit:
Change model padding

Changes modal padding to more evenly spread out content.

Remove modal__info class.

helrond reviewed (approved):
Looks nice!
